types/openlayers/index.d.ts

```diff
@@ -7132,6 +7132,12 @@ declare module ol {
         pointerEvent: ol.pointer.PointerEvent;
     }
 
+    interface DrawEvent {
```

Comment:
I don't see any `ol.DrawEvent` in the documentation. There is an `ol.interaction.DrawEvent` in versions before v3.19. But from v3.19, the `ol.interaction.DrawEvent` has been renamed to `ol.interaction.Draw.Event`. Is it the one you refer to? If it's the case, please change the definition.